bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
```

Comment:
Add "echos" just to tell what's happening so the logs are understandable

Reply:
done